tests/generic_config_updater/test_mmu_dynamic_threshold_config_update.py

```diff
@@ -0,0 +1,117 @@
+import logging
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert, pytest_require
+from tests.common.utilities import wait_until
+from tests.common.helpers.dut_utils import verify_orchagent_running_or_assert
+from tests.generic_config_updater.gu_utils import apply_patch, expect_op_success
+from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+from tests.generic_config_updater.gu_utils import create_checkpoint, delete_checkpoint, rollback_or_reload
+
+logger = logging.getLogger(__name__)
+
+READ_APPL_DB_TIMEOUT = 480
+READ_APPL_DB_INTERVAL = 20
+
```

Comment:
Can we add a pytest marker for topology and asic? Such as https://github.com/sonic-net/sonic-mgmt/blob/master/tests/generic_config_updater/test_pfcwd_interval.py#L10

Scripts without pytest.mark.topology are skipped in nightly test